src/lib/mongoBackend/mongoUpdateContext.cpp

```diff
@@ -68,8 +68,13 @@ HttpStatusCode mongoUpdateContext
     else
     {
         /* Process each ContextElement */
-        for (unsigned int ix= 0; ix < requestP->contextElementVector.size(); ++ix)
+        for (unsigned int ix = 0; ix < requestP->contextElementVector.size(); ++ix)
         {
+          //
```

Comment:
Nope, I find it 'nice' to start and end a longer comment like that. Surely, you must have seen this before ... ?